soc/nxp/imx/imx9/imx95/m7/soc.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/cache.h>
+
+void soc_late_init_hook(void)
+{
```

Comment:
use "#ifdef CONFIG_CACHE_MANAGEMENT" to include cache enable APIs? As I think we can change CACHE_MANAGEMENT to be "n", but it is "y" by default.

Reply:
sure, done